src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,14 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = [];
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state, action) => [...state, action.payload],
+    removeBook: (state, action) => [state.filter((item) => item.id !== action.id)],
+  },
+});
+
```

Comment:
- [ ] it would be better to export the `addBook` and `removeBook` as you did with the ` checkingStatus` action in order to complete the requirements